mirascope/core/openai/stream.py

```diff
@@ -61,30 +61,21 @@ def construct_call_response(self) -> OpenAICallResponse:
             raise ValueError(
                 "No stream response, check if the stream has been consumed."
             )
-        stream_tool_calls = self.message_param.get("tool_calls", [])
-        message = ChatCompletionMessage(
-            content=self.message_param.get("content", ""),
-            role=self.message_param["role"],
-        )
-        if stream_tool_calls:
-            tool_calls = []
-            for stream_tool_call in stream_tool_calls:
-                function_dict = stream_tool_call["function"]
-                tool_calls.append(
-                    ChatCompletionMessageToolCall(
-                        id=stream_tool_call["id"],
-                        function=Function(
-                            arguments=function_dict["arguments"],
-                            name=function_dict["name"],
-                        ),
-                        type="function",
-                    )
-                )
-            message.tool_calls = tool_calls
+        message = {
```

Comment:
this is unnecessary, just unroll the message_param

Reply:
as per our discussion, since there is an extra 'name' field this is fine.